test/parallel/test-crypto-stream.js

```diff
@@ -26,24 +26,24 @@ if (!common.hasCrypto)
 
 const assert = require('assert');
 const stream = require('stream');
-const util = require('util');
 const crypto = require('crypto');
 
 // Small stream to buffer converter
-function Stream2buffer(callback) {
-  stream.Writable.call(this);
+class Stream2buffer extends stream.Writable {
+  constructor(callback) {
+    super();
 
-  this._buffers = [];
-  this.once('finish', function() {
-    callback(null, Buffer.concat(this._buffers));
-  });
-}
-util.inherits(Stream2buffer, stream.Writable);
+    this._buffers = [];
+    this.once('finish', function() {
+      callback(null, Buffer.concat(this._buffers));
+    });
+  }
 
-Stream2buffer.prototype._write = function(data, encodeing, done) {
-  this._buffers.push(data);
-  return done(null);
-};
+  _write(data, encodeing, done) {
+    this._buffers.push(data);
+    return done(null);
+  }
+}
 
```

Comment:
(while we are here) how about eclosing the class definition inside the `common.hasFipsCrypto` block as it is used only if the conditional is met?

Reply:
I mean enclosing